### ISSUE TITLE
Defining literacy

### ISSUE THREAD

**Issue by agitter** [gitter-lab/ml-bio-workshop#49]
We will work here to define what we mean by "machine learning literacy".

**Comment by cmilica** [comment]
some important literacy questions could be:
How accurate is the model?
When is it most accurate?
When is it not so accurate?
What pipeline did it follow?

**Comment by agitter** [comment]
One set of literacy concepts would be those taught in machine learning courses.  At UW-Madison, that includes [CS 540](http://pages.cs.wisc.edu/~jerryzhu/cs540.html) and [CS 760](http://pages.cs.wisc.edu/~dpage/cs760/).  They are aimed at a different, more technical, audience, but there are some relevant ideas.  For instance, the first 760 lecture discusses when a trained model will generalize.  It also defines the course goals as:
> 1. Understand what a learning system should do
> 2. Understand how (and how well) existing systems work

The questions @cmilica posted above seem relevant to me.  The part about when a model is or is not accurate relates to generalization and the data distributions under which the model is expected to perform well.

Literacy likely also relates in part to being comfortable with some core vocabulary.  We could look for other machine learning glossaries or create our own that links to the definitions in our [guides](https://github.com/gitter-lab/ml-bio-workshop/tree/master/guide).

**Comment by dltreu** [comment]
Would "literacy" also include gaining the vocabulary and understanding of machine learning, sufficient to discuss a data-related research problem with a computationalist?

**Comment by dltreu** [comment]
What about identifying a biologist who has used machine learning or worked with a computationalist  and inviting them to give their perspective, review our materials, or somehow join the discussion?

**Comment by agitter** [comment]
> Would "literacy" also include gaining the vocabulary and understanding of machine learning

Yes, I'd say that is a major part of literacy.  We should think about ways to bring more attention to vocabulary and assess what terms are being retained.  Your prior suggestion to ask participants to explain concepts to a friend will help with this.

> What about identifying a biologist who has used machine learning

We should be able to find many examples of this.  Maybe I could contact some individuals who may be willing to talk to @cmilica about their experiences?

**Comment by cmilica** [comment]
I would be definitely ok with that. Before the meeting, it would be good to have specific questions we want to ask and focus on.

**Comment by agitter** [comment]
In our meeting today we created an initial list of terms.  @cmilica can create a new document where we discuss and finalize that list.
